.github/workflows/ci-test-ginkgo.yml

```diff
@@ -88,7 +88,7 @@ jobs:
         env:
           GOPATH: /home/runner/go
 
-      - uses: codecov/codecov-action@v3
+      - uses: actions/upload-artifact@v3 
```

Comment:
I think we might need to name the artifact like
https://github.com/codecov/example-actions-bundled/blob/2ab79565a2e2daa4d07857899b1089437461baac/.github/workflows/ci.yml#L28
Noting the name should use the matrix variable as well, else they will just be overwritten by each matrix run.